docs/docs/guides/build/assets/asset-selection-syntax.md

```diff
@@ -1,351 +1,440 @@
 
-A query includes a list of clauses. Clauses are separated by commas, except in the case of the `selection` parameter of the following methods. In these cases, each clause is a separate element in a list:
+In the **Dagster OSS UI**, the asset selection syntax is available on the assets overview page.
 
-- `define_asset_job`
-- `materialize`
-- `materialize_to_memory`
+In the **Dagster+ UI**, the asset selection syntax is available in:
 
-| Clause syntax         | Description                                                                                                                                                                                                                                                                                                               |
-| --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `ASSET_KEY`           | Selects a single asset by asset key. [See an example](#single-asset).                                                                                                                                                                                                                                                                                       |
-| `COMPONENT/COMPONENT` | Selects an asset key with multiple components, such as a prefix, where slashes (`/`) are inserted between components. [See an example](#multiple-key-components).                    |
-| `*ASSET_KEY`          | Selects an asset and all of its upstream dependencies. [See an example](#all-upstream).                                                                                                                                                                                                                             |
-| `ASSET_KEY*`          | Selects an asset and all of its downstream dependencies. [See an example](#all-downstream).                                                                                                                                                                                                                           |
-| `+ASSET_KEY`          | Selects an asset and one layer upstream of the asset. Including multiple `+`s will select that number of upstream layers from the asset. Any number of `+`s is supported. [See an example](#specific-upstream).    |
-| `ASSET_KEY+`          | Selects an asset and one layer downstream of the asset. Including multiple `+`s will select that number of downstream layers from the asset. Any number of `+`s is supported. [See an example](#specific-downstream). |
+- The Asset Catalog
+- The Alert Policy creation page (when creating an asset alert)
+- The Insights page
```

Comment:
The Asset Health overview page.

Reply:
And the same ones as OSS too (When reading it, it sounds like a disjoint set of surfaces from the OSS one)